sources/packages/web/src/components/institutions/request-a-change/CompletedOfferingChangeSummary.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <tab-container>
+    <body-header-container>
+      <template #header>
+        <body-header title="Applications" :recordsCount="applications?.count">
+          <template #subtitle>Outcome of the requested change</template>
+          <template #actions>
+            <v-text-field
+              density="compact"
+              label="Search name or application #"
+              variant="outlined"
+              v-model="searchCriteria"
+              @keyup.enter="searchApplicationOfferingChangeRecords"
+              prepend-inner-icon="mdi-magnify"
+              hide-details="auto"
+            >
+            </v-text-field>
+          </template>
+        </body-header>
+        <content-group>
+          <toggle-content :toggled="!applications?.count">
+            <v-data-table-server
+              :headers="CompletedOfferingChangeSummaryHeaders"
+              :items="applications?.results"
+              :items-length="applications?.count"
+              :loading="loading"
+              :items-per-page="DEFAULT_PAGE_LIMIT"
+              @update:options="paginationAndSortEvent"
+            >
+              <template #[`item.dateCompleted`]="{ item }">
+                {{ dateOnlyLongString(item.columns.dateCompleted) }}
+              </template>
+              <template #[`item.fullName`]="{ item }">
+                {{ item.columns.fullName }}
+              </template>
+              <template #[`item.studyStartPeriod`]="{ item }">
```

Comment:
Please make all the headers (Name, Study dates etc) in the datatable bold as per figma

Reply:
As per the current vuetify version, we don't have the ability to edit the datable header, but I noticed that lasted version does have the ability. I tried to upgrade the vuetify version to the latest 3.3.5. but there are some dependency conflicts, so, I don't want to add it to this ticket. 
![image](https://github.com/bcgov/SIMS/assets/77353155/39cc6a96-fd1d-473f-8dbd-7b7acf6ac95a)